sale_commission_product_criteria_country/tests/test_sale_commission_product_criteria_country.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2025 Alberto Martínez <alberto.martinez@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleCommissionProductCriteriaCountry(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.agent = cls.env.ref("sale_commission_product_criteria.demo_agent_rules")
+        cls.spain = cls.env.ref("base.es")
+        cls.france = cls.env.ref("base.fr")
+        cls.partner_spain = cls.env["res.partner"].create(
+            {
+                "name": "Test Spain",
+                "country_id": cls.spain.id,
+                "agent_ids": [(4, cls.agent.id)],
+            }
+        )
+        cls.partner_france = cls.env["res.partner"].create(
+            {
+                "name": "Test France",
+                "country_id": cls.france.id,
+                "agent_ids": [(4, cls.agent.id)],
+            }
+        )
+        cls.partner_no_country = cls.env["res.partner"].create(
+            {
+                "name": "Test B2B",
+                "agent_ids": [(4, cls.agent.id)],
+            }
+        )
+        cls.product = cls.env.ref("product.product_product_4")
+        cls.product_2 = cls.env.ref("product.product_product_1")
+        cls.commission = cls.env.ref(
+            "sale_commission_product_criteria.demo_commission_rules"
+        )
+        cls.com_item_1 = cls.env.ref(
+            "sale_commission_product_criteria.demo_commission_rules_item_1"
+        )
+        cls.com_item_1.country_id = False  # 10
```

Comment:
I guess that 10 means the comission it applies. Could you specify it, please?

Reply:
Changed